modules/apps/style-book/style-book-service/src/main/java/com/liferay/style/book/internal/frontend/css/variables/StyleBookScopedCSSVariablesProvider.java

```diff
@@ -104,6 +91,57 @@ public String getScope() {
 					return ":root";
 				}
 
+				private void _readCSSVariables(
+						Map<String, Object> cssVariables, JSONObject jsonObject)
+					throws JSONException {
+
+					Object defaultValue;
+
+					String variableType = jsonObject.getString("type");
+
+					if (Validator.isBlank(variableType)) {
+						throw new JSONException(
+							"Frontend token type not found");
+					}
+
+					FrontendToken.Type type = FrontendToken.Type.parse(
+						variableType);
+
+					if (type == FrontendToken.Type.BOOLEAN) {
+						defaultValue = jsonObject.getBoolean("defaultValue");
+					}
+					else if (type == FrontendToken.Type.INT) {
+						defaultValue = jsonObject.getInt("defaultValue");
+					}
+					else if (type == FrontendToken.Type.DOUBLE) {
```

Comment:
you can shorten code by below lines
if(Arrays.asList(FrontendToken.Type.values()).contains(type)){
	defaultValue = jsonObject.get("defaultValue");
}

Reply:
i fixed it. thank you